### DISCUSSION TITLE
Can I get the pageView count & active user statistics via API?

### DISCUSSION THREAD

**Discussion by weirongju** [open-metadata/OpenMetadata#18625] [Q&A] [answered]
Currently we could only get pageView & Active Users statistics by App Analytics page.
But there is limit information from the page, we cannot get the raw log to do more analysis.

Could we get the active users log & page view logs from API?

**Comment by Prajwal214** [comment, marked as answer]
@weirongju You can fetch the available APIs for App Analytics from Network Tab in the Browser console,
Example - 
`http://localhost:8585/api/v1/analytics/dataInsights/system/charts/name/percentage_of_data_asset_with_description`
